### DISCUSSION TITLE
Certain particles get stuck within SVG path

### DISCUSSION THREAD

**Discussion by motion-work** [matteobruni/tsparticles#1485] [Q&A] [answered]
In my config I have the following settings:

```js
polygon: {
  draw: {
    enable: true,
    stroke: {
      color: {
        value: 'rgba(255,255,255,0.2)',
      },
      width: 0.5,
      opacity: 0.2,
    },
  },
  enable: true,
  inline: {
    arrangement: 'equidistant',
  },
  move: {
    radius: 60,
    type: 'path',
  },
  scale: 2.5,
  type: 'inside',
  url:
    'https://res.cloudinary.com/delicious-cookie/image/upload/v1618317821/kaleido/Frame_10_2_jxq2lq.svg',
},
```

The particles should move within the defined svg path freely. But some of them get stuck randomly and wiggle.

[https://res.cloudinary.com/delicious-cookie/video/upload/v1618473154/kaleido/Screen_Recording_2021-04-15_at_09.51.37_ggrcsk.mov](https://res.cloudinary.com/delicious-cookie/video/upload/v1618473154/kaleido/Screen_Recording_2021-04-15_at_09.51.37_ggrcsk.mov)

**Comment by matteoums** [comment, marked as answer]
This is related to issue #1269, You can follow that to see when it's fixed
